lib/knackhq/client.rb

```diff
@@ -48,10 +48,9 @@ def records_by_page(object, page_number)
                      .records
                      .get(:params => { :page => page_number,
                                        :rows_per_page => 500 })
-                     .to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:records]
-      symbolize_hash_keys!(transform_hash_keys)
+      payload = payload_hash(hash_request)
+      return [] if payload.empty?
+      payload[:records]
     end
 
     def records_info(object)
```

Comment:
Appologies I was being misleading. 
The Knackhq client returns `records(object)` with Knackhqs' default of 25 rows per page. 

The records_info(object) is explicit in the amount of rows (500) returned and therefore can give an accurate amount of pages for other applications to consume.